BUILDING.md

```diff
@@ -45,6 +45,21 @@ Check that `cargo` is correctly installed by running
 cargo --version
 ```
 
+Next, head to https://rust-lang.github.io/rustup-components-history/ and find the most recent date that all the Tier1 platforms have a present rustfmt. For example: nightly-2019-03-04.
+
+Next, install that nightly version of rustfmt
+```
+rustup toolchain install nightly nightly-2019-03-04
+rustup component add rustfmt --toolchain nightly-2019-03-04 rustfmt
+```
+
+To run rustfmt on a cargo project
+```
+cargo +nightly-2019-03-04 fmt
```

Comment:
Wonder if we should encourage people to run `support/ci/rustfmt.sh` instead, since that similarly encapsulates handling of the specific nightly version (and is what we run in CI)

This also suggests that #6551 is something we should tackle soon, since that's a much more discoverable interface for these kind of things.

Reply:
I think https://github.com/habitat-sh/habitat/issues/6551 is a _really_ good idea and something we should prioritize sooner rather than later.  Having everything a developer needs to do driven through the `Makefile` is ideal imo, and a significant improvement in ergonomics over our current situation.